manifests/m/Microsoft/PCManager/1.2.6.4/Microsoft.PCManager.installer.yaml

```diff
@@ -0,0 +1,20 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=NVS0.CRLF.5-1-22621-1037.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Microsoft.PCManager
+PackageVersion: 1.2.6.4
+InstallerType: nullsoft
+Scope: machine
+InstallerSwitches:
+  Custom: /S
```

Comment:
```suggestion
  Custom: -activateapp=false
```

Reply:
I tried launching this version of the package with `-activateapp=false`, but it still provides a UI instead of a silent install. A silent install can be done normally using the `/S` parameter.

Reply:
Once the InstallerType is set to `nullsoft`, WinGet will be able to append `/S` by default.

Reply:
And `-activateapp=false` is used to prevent program from running after installation.